app/javascript/dashboard/components/widgets/conversation/ConversationBulkActions.vue

```diff
@@ -0,0 +1,288 @@
+              </div>
+            </li>
+          </ul>
+          <div v-else class="agent-confirmation-container">
+            <p>
+              {{
+                $t('BULK_ACTION.ASSIGN_CONFIRMATION_LABEL', {
+                  conversationCount: conversations.length,
+                })
+              }}
+              <strong>
+                {{ selectedAgent.name }}
+              </strong>
+            </p>
+            <div class="agent-confirmation-actions">
+              <woot-button
+                color-scheme="primary"
+                variant="smooth"
+                @click="goBack"
+              >
+                {{ $t('BULK_ACTION.GO_BACK_LABEL') }}
+              </woot-button>
+              <woot-button
+                color-scheme="primary"
+                variant="flat"
+                :is-loading="uiFlags.isUpdating"
+                @click="submit"
+              >
+                {{ $t('BULK_ACTION.ASSIGN_LABEL') }}
+              </woot-button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </transition>
+  </div>
+</template>
+
+<script>
+import Thumbnail from 'dashboard/components/widgets/Thumbnail.vue';
+import { mapGetters } from 'vuex';
+
+export default {
+  components: {
+    Thumbnail,
+  },
+  props: {
+    conversations: {
+      type: Array,
+      default: () => [],
+    },
+    allConversationsSelected: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  data() {
+    return {
+      showAgentsList: false,
+      query: '',
+      selectedAgent: null,
+    };
+  },
+  computed: {
+    ...mapGetters({
+      uiFlags: 'bulkActions/getUIFlags',
+      inboxes: 'inboxes/getInboxes',
+    }),
+    filteredAgents() {
+      return this.assignableAgents.filter(agent =>
+        agent.name.toLowerCase().includes(this.query.toLowerCase())
+      );
+    },
+    assignableAgents() {
+      return this.$store.getters['inboxAssignableAgents/getAssignableAgents'](
+        this.inboxes[0].id
+      );
+    },
+  },
+  mounted() {
+    this.$refs.selectAllCheck.indeterminate = true;
+    this.$store.dispatch('inboxAssignableAgents/fetch', {
+      inboxId: this.inboxes[0].id,
+    });
+  },
+  methods: {
+    assignAgent(agent) {
+      this.selectedAgent = agent;
+    },
+    selectAll(e) {
+      this.$emit('selectAllConversations', e.target.checked);
+    },
+    goBack() {
+      this.selectedAgent = null;
+    },
+    submit() {
+      this.$emit('assignAgent', this.selectedAgent);
+    },
+    resolveConversations() {
+      this.$emit('resolveConversations');
+    },
+  },
+};
+</script>
+
+<style scoped lang="scss">
+.flex-between {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+.bulk-action__container {
+  padding: var(--space-normal) var(--space-one);
+  background-color: var(--s-75);
+  position: relative;
+  box-shadow: 6px 3px 22px 9px rgb(181 181 181 / 25%);
```

Comment:
Use shadows from `app/javascript/shared/assets/stylesheets/shadows.scss`

Reply:
I needed a shadow which was upward, hence used a custom one. I can add this in the css file and use it as a variable.